src/lib/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestDetails/BorrowingRequestPatronLoan/BorrowingRequestPatronLoan.js

```diff
@@ -267,6 +269,13 @@ export default class BorrowingRequestPatronLoan extends React.Component {
           </Grid.Row>
         </Grid>
         <CreateLoan {...this.props} />
+
+        {!_isEmpty(brwReq.patron_loan) && (
+          <BorrowingRequestLoanExtension
+            patron_loan={brwReq.patron_loan}
```

Comment:
This is probably the point for which I left the previous comment `patronLoan={brwReq.patron_loan}`

Reply:
👀 thanks!